docs/user-guide/src/ironic/ironic-python-agent.md

```diff
@@ -0,0 +1,33 @@
+# ironic python agent (IPA)
+
+IPA is a service written in python that runs within a ramdisk. It provides remote access to  `ironic-conductor` and `ironic-inspector` services to perform various operations on the managed server. It also sends information about the server to `Ironic`.
+
+
```

Comment:
It may be useful to explain `ipa-downloader` and that we use RDO ramdisks by default.